includes/fbproductfeed.php

```diff
@@ -89,6 +89,10 @@ public function schedule_feed_generation() {
 		 * @since 1.11.0
 		 */
 		public function generate_feed() {
+			$profiling_logger = facebook_for_woocommerce()->get_profiling_logger();
+			$profiling_logger->start( 'generate_feed' );
+
+			do_action( 'qm/start', 'facebook_for_woocommerce__generate_feed' );
```

Comment:
@haszari shouldn't this Query Monitor start action have been removed from this PR?

Reply:
Yes it should have! 😱 I'll whip up a quick PR, thanks for catching!

Reply:
#1869 🧯 

Whoever reviews that - if it's all good feel free to merge for me 🤞